packages/camera/camera_platform_interface/lib/src/method_channel/method_channel_camera.dart

```diff
@@ -59,7 +59,7 @@ class MethodChannelCamera extends CameraPlatform {
   @override
   Future<List<CameraDescription>> availableCameras() async {
     try {
-      final cameras = await _channel
+      final List<Map<dynamic, dynamic>>? cameras = await _channel
```

Comment:
This happened in several places across the code. I think as @bparrishMines suggested we should enable a lint to enforce either always adding (or never adding) this type.

Reply:
Understood, I will leave it like this for now and when we decide on which version we will go for I will make sure everything is corrected.